packages/contracts-bedrock/deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export CONTRACT_ADDRESSES_PATH=/Users/maurelian/Projects/O/superchain-registry/superchain/extra/addresses/sepolia/pgn.json
```

Comment:
we should delete this file

Reply:
Yes, sorry about that. I dropped the extra commit.